conf/app.routes

```diff
@@ -1 +1,3 @@
 # microservice specific routes
+
+POST        /CDFPAY/v1/PostNewClaims         uk.gov.hmrc.cdsreimbursementclaim.controllers.PostNewClaimController.claim
```

Comment:
No versioning is required in the endpoint. We are not publishing an API, rather just consuming one. Change to just:

`POST     /claim`